apps/chat-e2e/src/tests/promptUsage.test.ts

```diff
@@ -65,7 +68,7 @@ dialTest(
         });
         promptToSelect = prompts[1];
         await sendMessage.messageInput.fillInInput('/');
-        await sendMessagePromptListAssertion.assertPromptListOptions(
+        await sendMessagePromptListAssertion.assertPromptListIncludesOptions(
           promptNames,
         );
         await sendMessagePromptListAssertion.assertPromptOptionOverflow(
```

Comment:
I think that we should parametrize that assertion to match specific option or at least rename the assertion, because you are checking that ".toBe(Overflow.ellipsis)" that is not obvious

Reply:
done